src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,38 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class StringCalculatorTest {
+
+    @DisplayName("빈문자열이 들어오면 0을 리턴")
+    @Test
```

Comment:
`@NullAndEmptySource`를 사용하면 어떨까요? 아래의 글을 참고하세요.
https://www.baeldung.com/parameterized-tests-junit-5

```suggestion
    @ParameterizedTest
    @NullAndEmptySource
```